add-on/chrome/bootstrap.js

```diff
@@ -533,15 +533,22 @@ var WindowListener = {
         this._hideBrowserSharingInfoBar();
 
         let box = gBrowser.getNotificationBox();
-        let pauseButtonLabel = this._getString(this._browserSharePaused ?
-                                               "infobar_button_resume_label" :
-                                               "infobar_button_pause_label");
-        let pauseButtonAccessKey = this._getString(this._browserSharePaused ?
-                                                   "infobar_button_resume_accesskey" :
-                                                   "infobar_button_pause_accesskey");
-        let barLabel = this._getString(this._browserSharePaused ?
-                                       "infobar_screenshare_paused_browser_message" :
-                                       "infobar_screenshare_browser_message2");
+
+        let getPausedUnpausedStrings = () => this._browserSharePaused ? {
+          label: this._getString("infobar_button_restart_label2"),
+          accesskey: this._getString("infobar_button_restart_accesskey"),
+          message: this._getString("infobar_screenshare_stop_sharing_message")
+        } : {
+          label: this._getString("infobar_button_stop_label2"),
+          accesskey: this._getString("infobar_button_stop_accesskey"),
+          message: this._getString("infobar_screenshare_browser_message2")
+        };
+        let stringObj = getPausedUnpausedStrings();
+
+        let pauseButtonLabel = stringObj.label;
+        let pauseButtonAccessKey = stringObj.accessKey;
+        let barLabel = stringObj.message;
+
```

Comment:
These 3 `let` declarations are unnecessary. Just directly use `stringObj` in the various places below.